bioimageio/spec/shared/common.py

```diff
@@ -33,9 +33,34 @@ def dump(self, data, stream=None, *, transform=None):
     yaml = MyYAML(typ="safe")
 
 
+try:
+    from tqdm import tqdm  # not available in pyodide
+except ImportError:
+
+    class tqdm:  # type: ignore
+        """no-op tqdm"""
+
+        def __init__(self, iterable: Optional[Iterable] = None, *args, **kwargs):
+            self.iterable = iterable
+
+        def __iter__(self):
+            if self.iterable is not None:
+                yield from self.iterable
+
+        def update(self, *args, **kwargs):
+            pass
+
+
 BIOIMAGEIO_CACHE_PATH = pathlib.Path(
     os.getenv("BIOIMAGEIO_CACHE_PATH", pathlib.Path(tempfile.gettempdir()) / "bioimageio_cache")
 )
+BIOIMAGEIO_NO_CACHE = bool(os.getenv("BIOIMAGEIO_NO_CACHE", "False").title())
```

Comment:
I don't think that this works as expected since `bool("False")` and `bool("0")` (the two most obvious values to set this to false via env vars) both evaluate to true.
I would also vote to rename this variable to `BIOIMAGEIO_USE_CACHE`, and set its default to true. This is much easier to understand than 'no_cahce" and default false; since the default is currently a double negation.

Reply:
very much on point! 👍

Reply:
Thanks for implementing this @FynnBe and I agree with @constantinpape for these points.

Reply:
yeah.. looking at it today I don't know what I was thinking yesterday^^